.eslintrc.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
         'packages/kbn-test/**/*',
         'packages/kbn-eslint-import-resolver-kibana/**/*',
         'x-pack/plugins/apm/**/*',
+        'x-pack/plugins/canvas/**/*',
```

Comment:
how come things like `src/*` aren't in here?  or other plugins like reporting?

Reply:
Prettier support is opt-in, we wanted to "take it slow" which basically just meant enabling it for one piece at a time when someone wanted to go through the pain of comiting the massive autofix pr.

Reply:
Does opting into Prettier here make the prettier config in the Canvas overrides redundant? It would be nice if that stuff was only configured once.